docs/v3.x/deployment/ibm-cloud.md

```diff
@@ -0,0 +1,347 @@
+
+```yml
+applications:
+  - name: 'My project'
+    routes:
+      - route: my-project.mybluemix.net
+    env:
+      FORCE_HTTPS: true
+      NODE_ENV: production
+      URL: https://my-project.mybluemix.net
+```
+
+#### 7. IBM Cloud proxy set-up
+
+Replace the `server.js` file with the following:
+
+`Path: ./config/server.js`.
+
+```js
+module.exports = ({ env }) => {
+  const myEnv = {
+    host: env('HOST', '0.0.0.0'),
+    port: env.int('PORT', 1337),
+    url: env('URL', ''),
+    admin: {},
+  };
+
+  return myEnv;
+};
+```
+
+#### 8. Create a IBM Cloud project
+
+With the first push the project is automatically created. This first run is going to fail due to database connection, but we're fixing it in the next steps.
+
+`Path: ./my-project/`
+
+```bash
+ibmcloud app push "My Project"
+```
+
+Your local development environment is now set-up and configured to work with IBM Cloud. You have a new Strapi project and a new IBM Cloud app ready to be configured to work with a database and with each other.
+
+#### 9. IBM Cloud Database set-up
+
+Below you will find database options when working with IBM Cloud. Please choose the correct database (e.g. PostgreSQL, MongoDB, etc.) and follow those instructions.
+
+:::: tabs
+
+::: tab PostgreSQL
+
+### IBM Postgres
+
+Follow these steps to deploy your Strapi app to IBM Cloud using **PostgreSQL**:
+
+#### 1. Create a [IBM Cloud Postgres service](https://cloud.ibm.com/catalog/services/databases-for-postgresql) for using Postgres
+
+The easiest way is to do it through the web management panel.
+
+#### 2. Retrieve database credentials
+
+Once the new service is ready, new credentials need to be created (`service credentials` section).
+
+A huge JSON file is generated but we are interested in the field called `connection.postgres.composed`. There you can find the values you need for the next step.
+
+```txt
+postgres://ibm_cloud_f999fe77_d44c_44f1_ad9a_d950111f381b:c6a1c55197f4855eb47d03401ddc5bfa9c8ba76472f420d7a1f968@00b111-888a-4f97-be11-e20a6517c0f9.6131b73286f34215871dfad7222b4f7d.databases.appdomain.cloud:31530/ibmclouddb?sslmode=verify-full"
+```
+
+(This url is read like so: `postgres:// USERNAME : PASSWORD @ HOST : PORT / DATABASE_NAME`)
+
+#### 3. Set environment variables
+
+Strapi expects a variable for each database connection configuration (host, username, etc.). So, from the url above, you have to set several environment variables in the IBM Cloud config:
+
+```bash
+ibmcloud app env-set "My Project" DATABASE_USERNAME ibm_cloud_f999fe77_d44c_44f1_ad9a_d950111f381b
+ibmcloud app env-set "My Project" DATABASE_PASSWORD c6a1c55197f4855eb47d03401ddc5bfa9c8ba76472f420d7a1f968
+ibmcloud app env-set "My Project" DATABASE_HOST 00b111-888a-4f97-be11-e20a6517c0f9.6131b73286f34215871dfad7222b4f7d.databases.appdomain.cloud
+ibmcloud app env-set "My Project" DATABASE_PORT 31530
+ibmcloud app env-set "My Project" DATABASE_NAME ibmclouddb
+```
+
+Please replace these above values with your actual values.
```

Comment:
Address the warning when setting variables from the CLI tool:
`TIP: Use 'cf restage DM-Strapi' to ensure your env variable changes take effect`

Reply:
This will need to be either `ic cf restage whatever` or `ibmcloud cf restage whatever`